src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -823,33 +823,29 @@ void Transformations::PostLpt() {
     // MLP & QKV fusion optimizations is focused on throughput, only enabled on AMX-bf16 & LLM serving use cases.
     auto can_use_amx_bf16 = dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core_amx) && (inferencePrecision == element::bf16);
     if (can_use_amx_bf16) {
-        auto has_paged_attention = op::util::has_op_with_type<ov::op::PagedAttentionExtension>(model);
-        if (has_paged_attention) {
-            CPU_REGISTER_PASS_X64(postLPTPassManager, MLPFusion);
-            CPU_SET_CALLBACK_X64(postLPTPassManager,
-                [](const_node_ptr &node) -> bool {
-                    std::string errorMsg;
-                    return node::LLMMLP::isSupportedOperation(node, errorMsg);
-                },
-                MLPFusion);
-        }
+        CPU_REGISTER_PASS_X64(postLPTPassManager, MLPFusion);
+        CPU_SET_CALLBACK_X64(postLPTPassManager,
+            [](const_node_ptr &node) -> bool {
+                std::string errorMsg;
+                return node::LLMMLP::isSupportedOperation(node, errorMsg);
+            },
+            MLPFusion);
 
         // Limitations: at least 3 workers are required for QKV fusion
         size_t concurrency = config.streamExecutorConfig.get_threads_per_stream();
         if (concurrency == 0)
             concurrency = parallel_get_max_threads();
         if (concurrency >= 3) {
-            if (has_paged_attention) {
-                CPU_REGISTER_PASS_X64(postLPTPassManager, QKVProjFusion);
-                CPU_SET_CALLBACK_X64(postLPTPassManager,
-                    [](const_node_ptr &node) -> bool {
-                        std::string errorMsg;
-                        return node::QKVProjection::isSupportedOperation(node, errorMsg);
-                    },
-                    QKVProjFusion);
-            }
+            CPU_REGISTER_PASS_X64(postLPTPassManager, QKVProjFusion);
```

Comment:
Given condition on PagedAttention is removed such optimization will be applied for all models including non LLMs (e.g. usual transformers). Have we verified performance on non LLM use-cases?

Reply:
I only verified `timm_vit_base_patch16_224` from model cache which turns out to contain no MLP or QKV pattern, it seems that some vit model has already fused QKV projection & MLP :
 - https://github.com/huggingface/pytorch-image-models/blob/26ed96b052c5ef442ed935c4da313e49e354f316/timm/models/davit.py#L293
 - https://github.com/huggingface/pytorch-image-models/blob/26ed96b052c5ef442ed935c4da313e49e354f316/timm/layers/mlp.py#L92

but there maybe some `vit` didn't fuse that, need validation team's help on that.